petric.py

```diff
@@ -165,7 +191,19 @@ def get_data(srcdir=".", outdir=OUTDIR, sirf_verbosity=0):
         penalty_strength = 1 / 700 # default choice
     prior = construct_RDP(penalty_strength, OSEM_image, kappa)
 
-    return Dataset(acquired_data, additive_term, mult_factors, OSEM_image, prior, kappa)
+    reference_image = STIR.ImageData(str(srcdir / 'reference_image.hv')) if (srcdir /
+                                                                             'reference_image.hv').is_file() else None
+    whole_object_mask = STIR.ImageData(str(srcdir /
```

Comment:
now need to read from `PETRIC_VOIs/*`. sorry

Reply:
why?